notify/jobs.py

```diff
@@ -11,7 +11,7 @@ def send_mail_to_attendees():
         .values_list("event_id", "customer_id")
         .distinct()
     )
-    "(event_id,customer_id)"
+    # "(event_id,customer_id)"
     for i in tickets:
```

Comment:
for such iterations prefer using singular terms , instead of i , eg: ticket, increases readability.

Reply:
okay